specificatie/genereervariant/openapi.yaml

```diff
@@ -1592,7 +1592,7 @@ paths:
       tags:
       - ingeschrevenpersonen
       description: Het ophalen de actuele partner-gegevens van een van een Ingeschreven
-        Persoon zoals die op de persoonslijst voorkomen. Een beëindigd huwelijk of
+        Persoon zoals die op de persoonslijst voorkomen. Een beeindigd huwelijk of
```

Comment:
beeindigd => beëindigd?

Reply:
Hier kan de resolver niet tegen. Dit leidt tot bijzondere karakters in de genereervariant. Het is dus kiezen tussen 2 kwaden......

Reply:
Volgens mij gaat het goed als je het resolven in GitHub laat gebeuren. Ik denk dat het te maken heeft met de combinatie Windows en de swagger-codegen-cli. Als je één van deze aanpast en commit op de niet genereervariant, dan kunnen we kijken of het dan via de GitHub actions goed gaat

Reply:
Ik heb dit inmiddels in de praktijk getest en dat werkt inderdaad goed als het resolve op Github plaatsvind. Ik ga de betreffende wijziging terugdraaien.